<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="selenium7" failures="1" tests="7" name="runners.CaseStudyRunner" time="13.882" errors="0" timestamp="2024-12-26T11:32:02 IST" skipped="0">
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="1.681"/>
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="0.499"/>
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="1.062"/>
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="6.731"/>
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="0.832">
    <failure message="expected [Article available] but found [Articles not available.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Article available] but found [Articles not available.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at stepDefinition.CaseStudyDef.message_should_display(CaseStudyDef.java:113)
at âœ½.Message should display(file:///C:/WorkSpace/BDDPOMCaseStudy/src/test/resources/feature/Conduit.feature:39)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="2.053"/>
  <testcase classname="runners.CaseStudyRunner" name="runScenario" time="1.024"/>
</testsuite> <!-- runners.CaseStudyRunner -->
